src/Menu.php

```diff
@@ -9,7 +9,7 @@
 
 class Menu extends View
 {
-    public $ui = 'menu';
+    public $ui = 'menu stackable';
```

Comment:
such behaviour is not typical, here is an example of mobile GH

![image](https://user-images.githubusercontent.com/2228672/214060629-4bff156c-636e-49e1-89e8-fc419c015f6b.png)

`stackable` might make UX between mobile and desktop too much different